c2corg_ui/templates/outing/view.html

```diff
@@ -36,15 +36,20 @@
 
 <%block name="moduleConstantsValues">
   <%
-      geometry = outing['geometry']['geom_detail'] \
-          if outing['geometry']['geom_detail'] else outing['geometry']['geom']
+      if outing.get('geometry'):
+          geometry = outing['geometry']['geom_detail'] \
+              if outing['geometry']['geom_detail'] else outing['geometry']['geom']
+      else:
+          geometry = None
   %>
   module.value('mapFeatureCollection', {
```

Comment:
If there is no geometry and you are not showing the map, you don't have to create the GeoJSON structure. You could simply use `module.value('mapFeatureCollection', null);` in that case.